src/util/redux.spec.js

```diff
@@ -0,0 +1,197 @@
+import assert from 'assert';
+import sinon from 'sinon';
+import {
+	assign,
+	isFunction,
+	upperCase
+} from 'lodash';
+
+import {
+	thunk,
+	getReduxPrimitives
+} from './redux';
+
```

Comment:
need a `describe` for each of the exported functions in `utils/redux.js` i only see a describe for `getReduxPrimitives`